### PR TITLE
Adds disclaimer to chatbot

### PR DESCRIPTION
**Pull request by BoDonkey**
## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*

## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly

## What kind of change does this PR introduce?
*(Check at least one)*

- [ ] Bug fix
- [ ] New feature
- [ ] Refactor
- [ ] Documentation
- [ ] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [ ] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [ ] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: docs/.vitepress/components/AposAISearchResults.vue [hunk 1]
```diff
@@ -86,7 +86,7 @@
           More questions? Have something to show off? Join the ApostropheCMS Discord
       </a>
     </div>
-    <div class="disclaimer">This feature is experimental and can make errors.</div>
+    <div class="disclaimer">This feature is experimental and can make errors. The Apostrophe team monitors questions so that we may improve results, do not submit sensitive information, api keys, credentials, etc...</div>
```
Comment (stuartromanek) [inline docs/.vitepress/components/AposAISearchResults.vue:89]:
```suggestion
    <div class="disclaimer">This feature is experimental and can make errors. To improve our service, the Apostrophe team reviews questions regularly. We kindly ask that you avoid sharing any sensitive information, such as API keys, credentials, or personal data.</div>
```

File: docs/.vitepress/components/AposAISearchResults.vue [hunk 2]
```diff
@@ -779,8 +779,11 @@ h2 {
 
 .disclaimer {
   font-size: 12px;
-  color: #a9a7a7;
+  color: #595858;
   text-align: center;
   margin-top: 10px;
+  max-width: 550px;
```
Comment (stuartromanek) [inline docs/.vitepress/components/AposAISearchResults.vue:785]:
```suggestion
  max-width: 570px;
```

**Review by stuartromanek — APPROVED** [review]
commit those suggestions and we're Gucci

**Comment by boutell** [comment]
Good suggested language

**Pull request merged**